PhysicsTools/NanoAOD/test/nano_cfg.py

```diff
@@ -8,6 +8,7 @@
 process.load("Configuration.StandardSequences.GeometryDB_cff")
 process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
 process.load('Configuration.StandardSequences.Services_cff')
+process.load("Configuration.StandardSequences.MagneticField_cff")
```

Comment:
What is this loading needed for?
Maybe this file could be simply left untouched in the 10_6 backport